### PR TITLE
Download CSV bug - handle fields with no accessor keys

### PR DESCRIPTION
**Pull request by djnunez-aot**
#2236 

-Allow columns without accesor keys to access their data

### CODE REVIEW

File: epictrack-web/src/components/shared/MasterTrackTable/utils.ts [hunk 1]
```diff
@@ -72,17 +72,51 @@ interface ExportToCsvOptions<T extends MRT_RowData> {
...
+    .map(
+      (staffMember: { first_name: string; last_name: string }) =>
+        `${staffMember.last_name} ${staffMember.first_name}`
+    )
+    .join("; ");
+}
+
+const customAccessors: { [key: string]: (row: any) => any } = {
+  "Responsible EPD": (row) =>
+    `${row.responsible_epd?.first_name} ${row.responsible_epd?.last_name}`,
```
Comment (jadmnsaad) [inline epictrack-web/src/components/shared/MasterTrackTable/utils.ts]: This is a utils file for the MasterTrackTable. Why do we have mention of Responsible EPD here and Work Lead here?

Comment (djnunez-aot) [reply epictrack-web/src/components/shared/MasterTrackTable/utils.ts]: lots of tables dont have accessor keys that are common and it causes issues with exporting

Comment (jadmnsaad) [reply epictrack-web/src/components/shared/MasterTrackTable/utils.ts]: but this component is general, it doesn't know or care about business logic like Responsible EPD or Work Lead. If this is the solution it shouldn't be here

Comment (djnunez-aot) [reply epictrack-web/src/components/shared/MasterTrackTable/utils.ts]: should i just allow passing custom accessors then into the export function

**Pull request merged**